packages/studio-base/src/panels/index.ts

```diff
@@ -38,21 +38,21 @@ const builtin: PanelInfo[] = [
   {
     title: `Diagnostics – Detail`,
     type: "DiagnosticStatusPanel",
-    description: "Display data for a given diagnostics node.",
+    description: "Display data for a given ROS diagnostics node.",
```

Comment:
There is no such thing as a "diagnostics node"; any node can publish to the /diagnostics topic.
```suggestion
    description: "Display ROS diagnostics data for a given node.",
```